README.md

```diff
@@ -1,3 +1,17 @@
 Ironic Prometheus Exporter
 ==========================
 
+
+Configuration
+=============
+
+After install the driver you will need to update the :ironic.conf: and add
+:file_dir: and :file_name: options
+
+```
+[oslo_messaging_notifications]
+driver = prometheus_exporter
+transport_url = fake://
```

Comment:
Pardon me if it's a stupid question, but can we merge `file_dir` and `file_name` into `transport_url`, like
```
transport_url=file:///tmp/ironic_prometheus_exporter/myfile.txt
```
?

Reply:
When I talked with Julia in the begin our idea was use something like this in the `transport_url`, but while testing I ended up with 
```
oslo_messaging.transport.DriverLoadFailure: Failed to load transport driver "file": No \'oslo.messaging.drivers\' driver found, looking for \'file\''
```
To use transport_url will be necessary create a Transport Driver

Reply:
Ugh. Okay, let's leave it as it is. Why do you split `file_dir` and `file_name` though?

Reply:
The reason I've split was just to let the driver verify if the directory exists and if not it will create

Reply:
You could do that with os.path.dirname on the full path.